modules/persistent-storage-csi-smb-cifs-create-pv.adoc

```diff
@@ -0,0 +1,192 @@
++
+[source, cli]
+----
+oc create -f <pv-file-name>.yaml <1>
+----
+<1> The name of the PVC YAML file.
++
+.Example PVC YAML file
++
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: <pvc-name> <1>
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: <storage-amount> <2>
+  volumeName: <pv-name> <3>
+  storageClassName: <sc-name> <4>
+----
+<1> The name of the PVC.
+<2> Storage request amount.
+<3> The name of the PV from the first step.
+<4> The name of the CIFS/SMB storage class that you created earlier (see _Prerequisites_).
+
+.. Ensure that the PVC was created and is in the "Bound" status by running the following command:
++
+[source, cli]
+----
+oc describe pvc <pvc-name> <1>
+----
+<1> The name of the PVC name that you created in the preceding step.
++
+.Example output
++
+[source,terminal]
+----
+Name:          pvc-test
+Namespace:     default
+StorageClass:  smb
+Status:        Bound <1>
+...
+----
+<1> PVC is in Bound status.
+
+. Create a deployment on Linux by running the following command with following example YAML file:
++
+[source, cli]
+----
+oc create -f <deployment-file-name>.yaml <1>
+----
+<1> The name of the deployment YAML file.
++
+.Example deployment YAML file
++
+[source, yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nginx
+  name: <deployment-name> <1>
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+      name: <deployment-name> <1>
+    spec:
+      nodeSelector:
+        "kubernetes.io/os": linux
+      containers:
+        - name: <deployment-name> <1>
+          image: quay.io/centos/centos:stream8
+          command:
+            - "/bin/bash"
+            - "-c"
+            - set -euo pipefail; while true; do echo $(date) >> <mount-path>/outfile; sleep 1; done <2>
+          volumeMounts:
+            - name: <vol-mount-name> <3>
+              mountPath: <mount-path> <2>
+              readOnly: false
+      volumes:
+        - name: <vol-mount-name> <3>
+          persistentVolumeClaim:
+            claimName: <pvc-name> <4>
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+    type: RollingUpdate
+----
+<1> The name of the deployment.
+<2> The volume mount path.
+<3> The name of the volume mount.
+<4> The name of the PVC created in the preceding step.
+
+. Check the setup by running the `df -h` command in the container:
++
+[source, cli]
+----
+oc exec -it <pod-name> -- df -h <1>
+----
+<1> The name of the pod.
++
+.Example output
++
+[source, terminal]
+----
+Filesystem            Size  Used Avail Use% Mounted on
+...
+/dev/sda1              97G   21G   77G  22% /etc/hosts
+//20.43.191.64/share   97G   21G   77G  22% /mnt/smb
+...
+----
++
+In this example, there is a _/mnt/smb_ directory mounted as a Common Internet File System (CIFS) filesystem.
```

Comment:
Since this is a filepath, do you think this should be in backticks instead? I think the [docs guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) count paths as stuff that should have backticks
```suggestion
In this example, there is a `/mnt/smb` directory mounted as a Common Internet File System (CIFS) filesystem.
```